package.json

```diff
@@ -132,7 +132,7 @@
   },
   "dependencies": {
     "@apollo/client": "^3.0.2",
-    "@colony/colony-js": "^3.0.0-alpha.15",
+    "@colony/colony-js": "^3.0.0-alpha.17",
```

Comment:
So I think we broke this when we removed the filters from colonyJS (all those `null` arguments)

Since now we broke the roles system in the app. 

I'm logged in with the creator of the colony, but I just get the recovery role for that user, and because of this, I get an error saying I can't upgrade my colony

![Screenshot from 2021-01-25 15-01-25](https://user-images.githubusercontent.com/1193222/105709516-7dd99080-5f1e-11eb-8f4c-5ed03424e8b1.png)

And these are the roles

![Screenshot from 2021-01-25 15-02-27](https://user-images.githubusercontent.com/1193222/105709527-8205ae00-5f1e-11eb-96ea-43c2c766b309.png)

Reply:
But don't worry about this, I'll look into colonyJS and try to fix it

Reply:
I have all the roles when I login but I still see the permission error because the colony can't be upgraded as it has the latest version. I think the UX has to be changed slightly (please, see my comment about it in my review summary and let me know what you both think).